apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2.java

```diff
@@ -135,7 +135,6 @@ private static ExplanationOfBenefit transformClaim(
     // CARR_CLM_CNTL_NUM              => ExplanationOfBenefit.extension
     TransformerUtilsV2.mapEobCommonGroupCarrierDME(
         eob,
-        claimGroup.getBeneficiaryId(),
```

Comment:
Guessing this wasn't used, which is why its being removed? (Should remove from the doc too)

Reply:
fixed javadoc to remove this param